### DISCUSSION TITLE
Make header image narrower?

### DISCUSSION THREAD

**Discussion by ternaus** [mmistakes/minimal-mistakes#2673] [Q&A] [answered]
The header image occupies the whole width of the screen. And the screens these days are very wide.

Is it possible to make it less wide?

**Comment by mmistakes** [comment, marked as answer]
No and yes.
The headers are designed to expand to fill the width of their parent.
If you want to alter that then you need to be comfortable with CSS and restyle them to not do that.

You could take [`.page__hero`](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_page.scss#L160), give it a `max-width` and then center it with `margin-left: auto;` and `margin-right: auto;`, similiar to what is done with the [`#main`content container](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_page.scss#L5-L20).

**Comment by ternaus** [comment]
Thanks!
